tower-balance/src/select/mod.rs

```diff
@@ -0,0 +1,28 @@
+use futures::Poll;
+use ordermap::IterMut;
+use std::hash::Hash;
+use tower::Service;
+
+use PollLoad;
+
+mod p2c;
+mod round_robin;
+
+pub use self::p2c::PowerOfTwoChoices;
+pub use self::round_robin::RoundRobin;
+
+/// A strategy for selecting nodes.
+pub trait Select {
```

Comment:
This is _definitely_ bikeshedding but I'm not sure if `Select` is the best name or not --- at the first glance, when I saw the `Select` field in `Balance`, I thought it was a `stream::Select` or `future::Select` or something. I think maybe a name like `SelectPolicy` or `SelectStrategy` would better communicate what this trait is (or even just `select::Policy`/`select::Strategy`?)

Reply:
Honestly, we may want to rename `Balance` -> `LoadBalancer`. That would free up the trait to be named `Balance`.

/disclaimer This is just an idea and not a change request.